### PR TITLE
feat: image preview esc handler #415

### PR DESCRIPTION
**Pull request by FrolYo**

### CODE REVIEW

**Review by crui3er — APPROVED** [review]

**Comment by alexyakunin** [comment]
@FrolYo `ImagePreviewModal.razor` is deleted in dev, maybe you'll resolve this faster.

File: src/dotnet/UI.Blazor/Components/ImagePreviewModal/image-preview-modal.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.image-preview-modal {
+    @apply p-0 border-none rounded-none gap-y-0 bg-transparent w-fit;
+    max-width: 90vw;
```
Comment (evgeniy-ac) [inline src/dotnet/UI.Blazor/Components/ImagePreviewModal/image-preview-modal.css:3]: max-h-[90vh] min-h-[90vh]

**Comment by FrolYo** [comment]
> @FrolYo `ImagePreviewModal.razor` is deleted in dev, maybe you'll resolve this faster.

fixed

File: src/dotnet/UI.Blazor/Components/ImagePreviewModal/image-preview-modal.css (hunk 1 above)
Comment (FrolYo) [reply src/dotnet/UI.Blazor/Components/ImagePreviewModal/image-preview-modal.css]: done, thx

**Review by evgeniy-ac — APPROVED** [review]

**Pull request merged**